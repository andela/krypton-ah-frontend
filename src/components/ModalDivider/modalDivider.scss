@import "../colors";
@import "../fonts";

.ui {
  &.horizontal {
    &.divider {
      font: {
        family: roboto;
        size: 0.875rem;
        weight: 100;
      };
      text-transform: none;
    }
  }
}
