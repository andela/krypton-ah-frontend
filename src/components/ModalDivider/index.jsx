import React from 'react';
import PropTypes from 'prop-types';
import { Divider } from 'semantic-ui-react';
import './modalDivider.scss';


function ModalDivider(props) {
  const { title } = props;
  return (
    <>
      <Divider horizontal>{ title }</Divider>
    </>
  );
}

ModalDivider.propTypes = {
  title: PropTypes.string.isRequired
};

export default ModalDivider;
