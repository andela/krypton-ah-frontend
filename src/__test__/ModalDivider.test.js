import React from 'react';
import { create } from 'react-test-renderer';
import ModalDivider from '../components/ModalDivider';

function checkDivider() {
  test('it should matches the snapshot', () => {
    const component = create(<ModalDivider />);
    expect(component.toJSON()).toMatchSnapshot();
  });
}

describe('Modal Divider', () => {
  checkDivider();
});
