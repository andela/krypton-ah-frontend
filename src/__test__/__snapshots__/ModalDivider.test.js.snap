// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal Divider it should matches the snapshot 1`] = `
<div
  className="ui horizontal divider"
>
  Or
</div>
`;
